cli/Cargo.toml

```diff
@@ -26,7 +26,7 @@ experimental = [
 ]
 
 health = ["reqwest", "serde_json"]
-
+biome-credentials = ["database-migrate-biome-credentials"]
```

Comment:
This should be placed in a separate list below these features, with a comment above it such as "# Aliases - these aliases exist to map libsplinter feature names to their CLI equivalents" (or similar).

Reply:
updated